src/applications/apps.js

```diff
@@ -2,8 +2,8 @@ import { ensureJQuerySupport } from '../jquery-support.js';
 import { isActive, isLoaded, isntLoaded, toName, NOT_LOADED, shouldBeActive, shouldntBeActive, isntActive, notSkipped } from './app.helpers.js';
 import { reroute } from 'src/navigation/reroute.js';
 import { find } from 'src/utils/find.js';
-import { toUnmountPromise } from 'src/applications/lifecycles/unmount.js';
```

Comment:
i moved the lifecycle files into a different directory since they don't always apply to applications anymore

Reply:
👍